probe-rs-tools/src/bin/probe-rs/cmd/cargo_embed/mod.rs

```diff
@@ -244,7 +244,7 @@ fn main_try(args: &[OsString], offset: UtcOffset) -> Result<()> {
         Err(e) => return Err(e.into()),
     };
 
-    let format = FormatOptions::default().to_format_kind(session.target());
+    let format = FormatKind::from(FormatOptions::default().to_format_kind(session.target()));
```

Comment:
This double convert seems odd 🤔

Reply:
You'll find that most of the library types had to be duplicated, because I didn't want to add Schema to half of the library, and also it's probably better if the RPC's types are independent of the libraries types. There's quite a bit of conversion needed, but in the long run we can still change the library without breaking the RPC.